android/app/build.gradle

```diff
@@ -88,8 +88,8 @@ android {
         applicationId "com.mendix.developerapp.mx10"
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
-        versionCode project.hasProperty('versionCode') ? Integer.parseInt(project.versionCode) : 10
```

Comment:
Why are we making it static here?
We will have to change it into two places every time